build/linux/installer/conf/kube.conf

```diff
@@ -1,270 +1,349 @@
-# Fluentd config file for OMS Docker - cluster components (kubeAPI)
     #fluent forward plugin
     <source>
-     type forward
+     @type forward
      port "#{ENV['HEALTHMODEL_REPLICASET_SERVICE_SERVICE_PORT']}"
      bind 0.0.0.0
      chunk_size_limit 4m
     </source>
 
     #Kubernetes pod inventory
     <source>
-     type kubepodinventory
-     tag oms.containerinsights.KubePodInventory
+     @type kube_podinventory
+     tag oneagent.containerInsights.KUBE_POD_INVENTORY_BLOB
      run_interval 60
-     log_level debug
+     @log_level debug
     </source>
 
     #Kubernetes Persistent Volume inventory
     <source>
-     type kubepvinventory
-     tag oms.containerinsights.KubePVInventory
+     @type kube_pvinventory
+     tag oneagent.containerInsights.KUBE_PV_INVENTORY_BLOB
```

Comment:
So these changes will go into AKS chart for our addon. How is this not a yaml change that will help easy rollback using image ?

Reply:
> So these changes will go into AKS chart for our addon. How is this not a yaml change that will help easy rollback using image ?
> as I mentioned earlier and also in the description, we will be updating config in kube.conf in omsagent.yaml in subsequenent releases. In main.sh, kube.conf & container.conf will be used by fluentd depending rs vs ds pod. This way, we can roll back to image if required at all.

Reply:
sorry, dont get it. kube.conf is coming from a configmap which is defined in the yaml. If you make changes to kube.conf, that needs to changed in yaml. So are you saying that the changes you have made now in kube.conf in this PR will not be released for stage2 ?

Reply:
> sorry, dont get it. kube.conf is coming from a configmap which is defined in the yaml. If you make changes to kube.conf, that needs to changed in yaml. So are you saying that the changes you have made now in kube.conf in this PR will not be released for stage2 ?
>  what I am saying, in stag2, fluentd will consume the kube.conf which will be available in /etc/fluent/kube.conf path and rather than coming from configmap in the yaml.  once that rolled out, in stage3, either we can keep the kubeconfig configmap in omsagent.yaml keep the sync with kube.conf or we can try the removing of the dependency of the kube.conf configmap dependency from omsagent yaml.